authors/apps/articles/views.py

```diff
@@ -28,11 +29,15 @@ class CreateArticlesAPIView(APIView):
     permission_classes = (IsAuthenticatedOrReadOnly, IsVerifiedUser,)
     serializer_class = ArticleSerializer
     renderer_classes = (ArticleJSONRenderer,)
+    pagination_class = LimitOffsetPagination
```

Comment:
Out of curiosity, why not use `PageNumberPagination`, which would make it easier to jump to a specific page as opposed to offset which jumps to a specific point in a "list"?